openedx/core/lib/course_tabs.py

```diff
@@ -0,0 +1,113 @@
+"""
+Tabs for courseware.
+"""
+from openedx.core.lib.api.plugins import PluginManager
+from xmodule.tabs import CourseTab, key_checker
+
+_ = lambda text: text
+
+
+# Stevedore extension point namespaces
+COURSE_VIEW_TYPE_NAMESPACE = 'openedx.course_tab'
+
+
+class TabManager(PluginManager):
```

Comment:
I'm thinking that TabManager is a bit too generic. It isn't really managing tabs. How about ``CourseTabPluginManager``?